Podverse/AppDelegate.swift

```diff
@@ -101,13 +101,17 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
 
     func skipBackwardEvent() {
-        PVMediaPlayer.shared.previousTime(seconds: 15)
-        PVMediaPlayer.shared.setPlayingInfo()
+        if let currentItem = pvMediaPlayer.avPlayer.currentItem {
+            pvMediaPlayer.goToTime(seconds: CMTimeGetSeconds(currentItem.currentTime()) - 15)
```

Comment:
Actually, I'm not sure we even need setPlayingInfo on time changes. I'll remove it for now and put it back if it turns out we need it.